Makefile

```diff
@@ -30,6 +30,9 @@ PLATFORM=linux/ppc64le
 
 .EXPORT_ALL_VARIABLES:
 
+.PHONY: all
+all: verify test bin/ibm-powervs-block-csi-driver image
```

Comment:
but where do we use? do you want to running everything in one shot? Its good to run these things in parallel for better error reporting,

Reply:
During the presubmit build from prow, ibm-vpc-block-csi-driver runs all these targets.
If not required, we may run only target build.

Reply:
I feel running them individual will be a better idea I feel to better reporting and coverage, e.g: https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1543.

Reply:
Makes Sense. PR can be closed.